bin/convertToIcns

```diff
@@ -48,23 +47,12 @@ trap cleanUp EXIT
 mkdir -p "${TEMP_DIR}"
 mkdir -p "${ICONSET}"
 
-PNG_PATH="${TEMP_DIR}/icon.png"
-${CONVERT_TO_PNG} "${SOURCE}" "${PNG_PATH}"
+sips -s format tiff "$SOURCE" --out "$TEMP_DIR/icon_128.tiff" -z 128 128 2>&1 > /dev/null & 
+# sips hasn't always finish processing by the time the process exits,  so make
+# sure we wait till it finishes
+while ps ax | grep -v grep | grep -q $! ; do sleep .1; done
+tiff2icns "$TEMP_DIR/icon_128.tiff" "$DEST" 2>&1 > /dev/null
```

Comment:
Finally, it’s probably a bad idea to redirect output to `/dev/null`, here. `tiff2icns` has no output when it succeeds, so errors should likely not be ignored.